otherlibs/systhreads/Makefile

```diff
@@ -19,9 +19,9 @@ CAMLYACC ?= ../../boot/ocamlyacc
 
 ROOTDIR=../..
 CAMLC=$(CAMLRUN) $(ROOTDIR)/ocamlc -nostdlib \
-      -I $(ROOTDIR)/stdlib -I $(ROOTDIR)/otherlibs/unix
+      -I $(ROOTDIR)/stdlib -I $(ROOTDIR)/otherlibs/unix -I $(ROOTDIR)/otherlibs/win32unix
```

Comment:
How about something along these lines to not include the path on other non-windows systems.

```
ifeq ($(TOOLCHAIN),mingw)
INCLUDE_WINPATH=-I $(ROOTDIR)/otherlibs/win32unix
else
INCLUDE_WINPATH=
endif
 ROOTDIR=../..
 CAMLC=$(CAMLRUN) $(ROOTDIR)/ocamlc -nostdlib \
      -I $(ROOTDIR)/stdlib -I $(ROOTDIR)/otherlibs/unix $(INCLUDE_WINPATH)
```